packages/website/src/bin/default-config.js

```diff
@@ -1,10 +1,15 @@
 const identityFunc = require('lodash.identity');
 
 module.exports = {
-  docs: './docs',
+  docs: {
+    path: './docs',
+    name: 'Docs',
+    description: 'This is a sample documentation for the website.',
```

Comment:
Can we change this description to "View the documentation for this project" - so if the default isn't changed it's less of a footgun?